pkg/collector/corechecks/cluster/helm/helm_test.go

```diff
@@ -194,9 +194,11 @@ func TestRun(t *testing.T) {
 			check := factory().(*HelmCheck)
 			check.runLeaderElection = false
 
-			k8sClient := fake.NewSimpleClientset(kubeObjects...)
-			sharedK8sInformerFactory := informers.NewSharedInformerFactory(k8sClient, time.Minute)
-			err := check.setupInformers(sharedK8sInformerFactory)
+			check.informerFactory = informers.NewSharedInformerFactory(
+				fake.NewSimpleClientset(kubeObjects...),
+				time.Minute,
```

Comment:
could you remember me, what are the purpose of this `time.Minute` duration? is it for resync? timeout?